examples/launch_standalone_bayesian_optimization.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import logging
+import sys, os
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
Can you try to remove this one? It seems that the error you got was an unused import, but this can be solved by using `#noqa` flag, for instance:

```python
# from syne_tune.optimizer.baselines import PASHA, BORE  # noqa: F401
```

Reply:
We need a manual configuring of the file path, since we are not running setup.py, it cannot identify syne_tune in the directory. The error message I've got when the line is omitted : https://github.com/jjaeyeon/syne-tune/runs/6939346421?check_suite_focus=true

Reply:
Update: I've remove the line and add PYTHONPATH in the workflow